letsdrill.rb

```diff
@@ -1,13 +1,21 @@
 def get_letter_grade(integer)
- 
-  #Put your code here!
-
+  grades = {
+    "A" => 90..100,
+    "B" => 80..89,
+    "C" => 70..79,
+    "D" => 60..69,
+    "F" => 0..59
+  }
+  
+  grades.each do |k,v|
+    if v === integer
+      return k
```

Comment:
I dig what you're trying to do here, but I don't think it turned out better than `if`/`elsif` or `case`.

What about something like:

```ruby
GRADES = %w( F F F F F F D C B A )

def get_letter_grade(integer)
  GRADES[integer / 10]
end
```

or something like that?

I think everybody else settles for `if`/`elsif` or `case` on this though. 😁